mdtk/tests/test_midi.py

```diff
@@ -0,0 +1,199 @@
+import pandas as pd
+import os
+import pretty_midi
+import shutil
+
+import mdtk.midi as midi
+
+USER_HOME = os.path.expanduser('~')
+TEST_CACHE_PATH = os.path.join(USER_HOME, '.mdtk_test_cache')
+MIDI_PATH = f"mdtk{os.path.sep}tests{os.path.sep}test.mid"
+
+def test_df_to_csv():
+    notes = []
```

Comment:
You can:

```
notes = {
    'onset': [0, .3, .3],
    'track': [1, 1, 2],
    'pitch': [42, 40, 56],
    'dur': [1.5, 1.07, 1.11]
}
df = pd.DataFrame(notes)
```
if you'd like to use fewer lines. Your call - your way is perhaps clearer.

Reply:
I'll stick with mine here for now. If it gets much longer, I can change this.

Reply:
No problem